docker/smqtk_iqr_playground/Dockerfile

```diff
@@ -112,6 +114,10 @@ ENV LC_ALL=C.UTF-8 \
 RUN ln -s /usr/share/postgresql-common/pg_wrapper /usr/local/bin/pg_ctl \
  && ln -s /usr/share/postgresql-common/pg_wrapper /usr/local/bin/postgres
 
+# TODO: figure out a better way to do this
+RUN ln -s /usr/local/lib/python3.6/dist-packages/smqtk_iqr/web/search_app/templates/base.html \
+          /usr/local/lib/python3.6/dist-packages/smqtk_iqr/web/search_app/modules/iqr/templates/base.html
+
```

Comment:
For some reason this symbolic link wasn't showing up in the docker container so I added it here manually. Maybe there is a better way to do this or another reason that symbolic link is disappearing.

Reply:
Yeah, the symlink exists on my host system and when I copy the smqtk_iqr directory to /smqtk/source in the container. It looks like it doesn't get copied into dist-packages whenever I do pip install. I believe the original mono-repo used cmake to build, instead of pip, so maybe there is some difference there?

Reply:
There could be a difference. I would prefer to not use kludges where possibly, so maybe take a little more time to investigate this, otherwise we can use that kludge with a very obvious comment as to *why* we put that in there (like `# !!! NOTE THIS KLUDGE !!!` or something).

Reply:
From what I've found online, symlinks aren't supported by pip currently:
https://github.com/pypa/pip/issues/5919
https://discuss.python.org/t/symbolic-links-in-wheels/1945/3

Looks like after the mono-repo cmake build, `base.html` is substituted for the symlink directly, so no issues there.

Reply:
OK, just use the `RUN ln -s ...` in the Dockerfile then, with a comment amounting to what you just said above.